hr_employee_ssn/views/hr_employee_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+      Copyright (C) 2018 Brainbean Apps (https://brainbeanapps.com)
+      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+    -->
+
+    <record id="view_employee_form" model="ir.ui.view">
+        <field name="name">hr.employee.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="groups_id" eval="[(6, 0, [ref('hr.group_hr_user') ])]"/>
```

Comment:
Hello @alexey-pelykh 

The [page](https://github.com/odoo/odoo/blob/12.0/addons/hr/views/hr_views.xml#L80) where you add this fields already have this group,

Could you remove this?

Regards